src/main/java/org/webshar/hrms/controller/EmployeeController.java

```diff
@@ -39,20 +39,20 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(EmployeeController.class);
 
-  @GetMapping(value = "/services/api/web/hrms/employees/{employee_id}")
-  public ResponseEntity<Response> getEmployeeById(
+  @GetMapping(value = "/{employee_id}")
```

Comment:
As per REST endpoints convention words need to be separated by -(hyphen) in the URL path.
E.g. `/employee-id` instead of `employee_id`